modules/upgrade/pages/proxy-uyuni.adoc

```diff
@@ -23,16 +23,16 @@ opensuse_leap15_2-uyuni-client \
 uyuni-proxy-stable-leap-152
 ----
 . Fully synchronize all channels with [command]``spacewalk-repo-sync``.
-    In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
+  In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
 
 
 
 [[uyuni-202007-channeldupes]]
```

Comment:
There's still this 202007 thing here, right?

Reply:
yes, I kept it that way. It is just a label or ID.  I usually change those things only if we do a general cleanup (of if I think it is too confusing).